visualization/app/codeCharta/state/store/metricData/edgeMetricData/edgeMetricData.reducer.ts

```diff
@@ -92,13 +92,17 @@ function getMetricDataFromMap() {
 
 	for (const [edgeMetric, occurrences] of nodeEdgeMetricsMap) {
 		let maximumMetricValue = 0
+		let minimumMetricValue = Number.MIN_SAFE_INTEGER
 		for (const value of occurrences.values()) {
 			const combinedValue = value.incoming + value.outgoing
 			if (combinedValue > maximumMetricValue) {
 				maximumMetricValue = combinedValue
 			}
+			if (combinedValue <= minimumMetricValue) {
```

Comment:
Why do you use `if`? What about `else` or `else if`? Could you please explain what this reducer is doing?